<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>681</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Zeitaku/Downloads/pngegg (3).png</normaloff>C:/Users/Zeitaku/Downloads/pngegg (3).png</iconset>
   </property>
  </widget>
  <widget class="QTreeView" name="treeView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>51</y>
     <width>251</width>
     <height>401</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="selected_file_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>681</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Now Editing:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_file_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string comment="Creates a new file. (.txt)"/>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_folder_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string comment="Creates new folder."/>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="text">
    <string>New Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string comment="Deletes selected file."/>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="text">
    <string>Delete File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string comment="Logout and saves your works to cloud."/>
   </property>
   <property name="toolTipDuration">
    <number>2</number>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sync_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string comment="Sync your works with the cloud"/>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="text">
    <string>Sync</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
